src/Common/src/System/Numerics/Hashing/HashHelpers.cs

```diff
@@ -6,12 +6,14 @@ namespace System.Numerics.Hashing
 {
     internal static class HashHelpers
     {
+        public static int RandomSeed { get; } = Guid.NewGuid().GetHashCode();
```

Comment:
Not trying to prevent the programmer from deriving the seed - don't have issue with that. 

It's to reduce the predictability of the seed; `CoCreateGuid` [isn't completely unpredictable](https://en.wikipedia.org/wiki/Globally_unique_identifier#Algorithm).